spring-integration-webflux/src/main/java/org/springframework/integration/webflux/outbound/WebFluxRequestExecutingMessageHandler.java

```diff
@@ -193,11 +202,22 @@ public void setPublisherElementTypeExpression(Expression publisherElementTypeExp
 		this.publisherElementTypeExpression = publisherElementTypeExpression;
 	}
 
+	public void setAttributeVariablesExpression(Expression attributeVariablesExpression) {
+		Assert.notNull(attributeVariablesExpression, "'attributeVariablesExpression' must not be null");
+		this.attributeVariablesExpression = attributeVariablesExpression;
+	}
+
 	@Override
 	public String getComponentType() {
 		return (isExpectReply() ? "webflux:outbound-gateway" : "webflux:outbound-channel-adapter");
 	}
 
+	@Override
+	protected final void doInit() {
+		BeanFactory beanFactory = getBeanFactory();
+		this.evaluationContext = ExpressionUtils.createStandardEvaluationContext(beanFactory);
```

Comment:
You can just have a single line here: doesn't look like there is a reason in extracting `beanFactory` variable.

Reply:
beanFactory variable has been removed and code has been updated to single line